src/services/users.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+let users = [];
+
+function getAllUsers() {
+  return users;
+}
+
+function findUserById(userId) {
+  const foundUser = users.find(({ id }) => id === userId);
+
+  return foundUser || null;
+}
+
+function createUser(name) {
+  const newUserId = users.length + 1;
```

Comment:
Instead of using the users' length to generate the new user ID, use a variable to store the next available ID and increment it every time a new user is created. This will avoid potential ID conflicts when users are removed.
Make function for it or use `uuid` lib for it